src/backend/gl/src/info.rs

```diff
@@ -171,6 +171,7 @@ pub struct PrivateCaps {
     pub buffer_storage_supported: bool,
     pub clear_buffer_supported: bool,
     pub frag_data_location_supported: bool,
+    pub is_embedded: bool,
```

Comment:
I don't disagree, I was using the same naming scheme as elsewhere in the code:
[src/backend/gl/src/info.rs#L24](https://github.com/gfx-rs/gfx/blob/6a300e46a239bfbac8394d500759f7210b692568/src/backend/gl/src/info.rs#L24)
[src/backend/gl/src/tex.rs#L559](https://github.com/gfx-rs/gfx/blob/38a6aa384794f77ff11361b6ca507064b20a88e7/src/backend/gl/src/tex.rs#L559)

There are enough silly little differences between GL and GLES that an embedded flag to cover all bases may be preferable.